### ISSUE TITLE
VPN client fails to start macOS

### ISSUE THREAD

**Issue by jdknives** [skycoin/skywire#378]
**Describe the bug**
When running the VPN client on macOS, the following error is thrown: 

```
[2020-05-28T19:18:22.954344+08:00] INFO (STDERR) [proc:vpn-client:b789b31decb944b18304f672129af205]: time="2020-05-28T19:18:22+08:00" level=fatal msg="Error creating VPN client" error="error getting Dmsg discovery IP: env arg ADDR_DMSG_DISC is not provided"
```

A second error was thrown on a different machine: 

```
2020/05/29 14:10:14 [ERR] yamux: keepalive failed: i/o deadline reached
[2020-05-29T14:10:14+03:00] WARN [snet.dmsgC]: Stopped accepting streams. error="keepalive timeout" session=035915c609f71d0c7df27df85ec698ceca0cb262590a54f732e3bbd0cc68d89282
[2020-05-29T14:10:14+03:00] INFO [snet.dmsgC]: Session stopped. error="failed to serve dialed session to 035915c609f71d0c7df27df85ec698ceca0cb262590a54f732e3bbd0cc68d89282: keepalive timeout"
[2020-05-29T14:10:14+03:00] INFO [snet.dmsgC]: Updating entry. entry=    version: 0.0.1
```

**Environment information:**

- OS: e.g. macOS 

**Steps to Reproduce**

1. Checkout `develop`
2. Use the attached config. 
3. Start visor with sudo privileges.
4. Establish tp to remote passed to VPN client
5. Start VPN client

```
{
	"version": "v1.0.0",
	"sk": "c074593bc6b98ce797ce6242cfba5f0912f9c46f60533c673518928df4ebf583",
	"pk": "021a9935d60b81d248d25104c5f8ee9a2a26bf545d21e3f3d481b36269ee6f8718",
	"dmsg": {
		"discovery": "http://dmsg.discovery.skywire.cc",
		"sessions_count": 1
	},
	"dmsgpty": {
		"port": 22,
		"authorization_file": "./dmsgpty/whitelist.json",
		"cli_network": "unix",
		"cli_address": "/tmp/dmsgpty.sock"
	},
	"stcp": {
		"pk_table": {
			"03302d940c9b95afa43897df7626c4773624a5b3185fa65ccb88fb9302f2d154c3": "172.104.103.48:7777"
		},
		"local_address": ":7777"
	},
	"transport": {
		"discovery": "http://transport.discovery.skywire.cc",
		"log_store": {
			"type": "file",
			"location": "./transport_logs"
		},
		"trusted_visors": null
	},
	"routing": {
		"setup_nodes": [
			"026c5a07de617c5c488195b76e8671bf9e7ee654d0633933e202af9e111ffa358d"
		],
		"route_finder": "http://routefinder.skywire.cc",
		"route_finder_timeout": "10s"
	},
	"uptime_tracker": {
		"addr": "http://uptime-tracker.skywire.cc"
	},
	"launcher": {
		"discovery": {
			"update_interval": "30s",
			"proxy_discovery_addr": "http://proxy.discovery.skywire.cc"
		},
		"apps": [
			{
				"name": "skychat",
				"args": [
					"-addr",
					":8001"
				],
				"auto_start": true,
				"port": 1
			},
			{
				"name": "skysocks",
				"auto_start": true,
				"port": 3
			},
			{
				"name": "skysocks-client",
				"auto_start": false,
				"port": 13
			},
			{
				"name": "vpn-server",
				"auto_start": false,
				"port": 44
			},
			{
				"name": "vpn-client",
				"args": [
					"-srv",
					"02c61b06ad757c50ffe31e9d3e7873bb8cb990ec4c6995c5c6c1590601728bc413",
					"-sk",
					"c074593bc6b98ce797ce6242cfba5f0912f9c46f60533c673518928df4ebf583"
				],
				"auto_start": false,
				"port": 43
			}
		],
		"server_addr": "localhost:5505",
		"bin_path": "./apps",
		"local_path": "./local"
	},
	"hypervisors": null,
	"cli_addr": "localhost:3435",
	"log_level": "info",
	"shutdown_timeout": "10s",
	"restart_check_delay": "1s"
```

**Comment by jdknives** [comment]
Another issue pops up right now since the service discovery has been deployed on test deployment. 

```
[2020-06-05T21:26:47+08:00] INFO [visor:startup]: Startup complete!
panic: non-positive interval for NewTicker

goroutine 101 [running]:
time.NewTicker(0x0, 0x0)
	/usr/local/go/src/time/tick.go:23 +0x147
github.com/SkycoinProject/skywire-mainnet/pkg/servicedisc.(*HTTPClient).UpdateLoop(0xc0000c7700, 0xcfd400, 0xc0001b9040, 0x0)
	/root/go/src/github.com/SkycoinProject/skywire-mainnet/pkg/servicedisc/client.go:211 +0xcc
github.com/SkycoinProject/skywire-mainnet/pkg/app/appdisc.(*proxyUpdater).Start.func1(0xc000298600, 0xcfd400, 0xc0001b9040)
	/root/go/src/github.com/SkycoinProject/skywire-mainnet/pkg/app/appdisc/discovery_updater.go:55 +0x4b
created by github.com/SkycoinProject/skywire-mainnet/pkg/app/appdisc.(*proxyUpdater).Start
	/root/go/src/github.com/SkycoinProject/skywire-mainnet/pkg/app/appdisc/discovery_updater.go:54 +0x11f
```

This only appears on VPN server side for now and only for VPN, but not for socksproxy.

**Comment by jdknives** [comment]
Fixed by #390
